bridge/src/worker.rs

```diff
@@ -30,6 +30,13 @@ pub enum WorkerError {
     ChannelClosed(),
 }
 
+#[derive(Debug, Clone)]
+pub struct WorkerConfig {
+    pub namespace: String,
+    pub task_queue: String,
+    pub build_id: String,
+}
+
```

Comment:
Same as with the client options, just use Core's types, otherwise this will eventually be exactly equivalent to them.